packages/ui-tooltip/src/Tooltip/__examples__/Tooltip.examples.tsx

```diff
@@ -21,35 +21,34 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+
 import React from 'react'
+import type { TooltipProps } from '../props'
 
 export default {
-  maxExamplesPerPage: 50,
-  propValues: {},
-  // @ts-expect-error ts-migrate(6133) FIXME: 'props' is declared but its value is never read.
-  getComponentProps: (props) => {
+  maxExamplesPerPage: 30,
+  propValues: {
+    children: [
+      <a key={1} href="/">
+        Tooltip
+      </a>
+    ]
+  },
+  getComponentProps: (_props: TooltipProps) => {
     return {
       renderTip: 'Hello world',
       isShowingContent: true,
-      children: <a href="/">Tooltip</a>
-    }
+      defaultIsShowingContent: true,
+      placement: 'bottom center'
+    } as const
   },
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any' type.
-  getExampleProps: (props) => {
+  getExampleProps: (props: TooltipProps) => {
     return {
-      dir: props.dir,
+      dir: (props as any).dir,
       as: 'div',
-      width: '100%',
-      height: '5rem',
       margin: 'large small',
-      padding: 'large small',
+      padding: 'x-large',
       textAlign: 'center'
-    }
-  },
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any' type.
-  filter: (props) => {
-    if (!props.defaultIsShowingContent) return true
-
-    return false
+    } as const
   }
```

Comment:
no actual changes here, just some refactoring

Reply:
it did change the examples in the VRT, are those valid changes?